pennylane/plugins/expt_tensornet.py

```diff
@@ -337,7 +345,18 @@ def _state(self):
 
     def reset(self):
         """Reset the device"""
-        self.__init__(self.num_wires)
+        self._nodes = []
+        self._edges = []
+
+        zero_state = np.zeros([2] * self.num_wires)
+        zero_state[tuple([0] * self.num_wires)] = 1.0
+        zero_state = self.array(zero_state, dtype=self.C_DTYPE)
+
+        # TODO: since this state is separable, can be more intelligent about not making a dense matrix
+        self._state_node = self._add_node(
+            zero_state, wires=tuple(w for w in range(self.num_wires)), name="AllZeroState"
+        )
```

Comment:
One approach for the TODO comment here would be something along the lines of:
```python
self._state_node = []
for wire in range(len(self.num_wires)):
        zero_state = np.zeros([2])
        zero_state[tuple([0])] = 1.0
        zero_state = self.array(zero_state, dtype=self.C_DTYPE)
        self._state_node.append(self._add_node(
            zero_state, wires=wire, name="ZeroStateWire" + str(wire)
        ))
```
In this case, however, ``self._state_node`` would be a list of ``tn.Node`` objects. @co9olguy do you think this could work, or do you know if there would be a way of storing the Kronecker product of the zero states?